test/server/policies.ts

```diff
@@ -0,0 +1,71 @@
+import { expect, test } from 'vitest'
+import { app } from './utils'
+
+test('policy list filtering', async () => {
+  const res = await app.inject({
+    method: 'GET',
+    path: '/policies?limit=5',
+  })
+  expect(res.statusCode).toBe(200)
+  const policies = res.json()
+  expect(Array.isArray(policies)).toBe(true)
+  expect(policies.length).toBeLessThanOrEqual(5)
+})
+
+test('policy list with specific included schema', async () => {
```

Comment:
we need to create some in public and in other schema(s) first (we prob have only 1)